test/integration/pkg/generated/openapi_generated.go

```diff
@@ -188,6 +188,24 @@ func schema__testdata_listtype_Item(ref common.ReferenceCallback) common.OpenAPI
 							Format: "int32",
 						},
 					},
+					"a": {
```

Comment:
I think there was no error because the bot in this repo doesn't rerun test before submit-- CI would fail for https://github.com/kubernetes/kube-openapi/pull/106, but the bot merges it because it sees lgtm and approval labels.

Reply:
Thanks for explaining, I was wondering what happened.

@BenTheElder how do we make CI rerun before merging?

Reply:
prow jobs (like kubernetes presubmits) will do this, as far as we know travis doesn't support any way for us to trigger it. if they get an API for that the merge robot could be extended to use it, or the presubmits in this repo could be converted to prowjobs in test-infra.

Reply:
Thanks for the explanation, I filed #114.